vizro-core/src/vizro/static/css/bootstrap_overwrites.css

```diff
@@ -113,6 +113,19 @@ label[for="theme-selector"] {
   padding: 0;
 }
 
+.tooltip-inner {
+  max-width: 500px !important;
+}
+
+/* TODO NOW: fix light theme readability */
+.tooltip-inner * {
+  color: var(--text-primary-inverted);
+}
+
+.tooltip-inner a:hover {
+  color: var(--text-primaryHover-inverted);
+}
```

Comment:
@huong-li-nguyen something like this is needed to ensure that the link doesn't disappear when you hover over it. I wasn't sure the right variable to go for but this sounded like it might be the right choice?